examples/calendar-widget/index.html

```diff
@@ -41,7 +41,7 @@ <h3>Dates</h3>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
   <script src="https://unpkg.com/BaremetricsCalendar@1.0.11/public/js/Calendar.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/algoliasearch@3.32.0"></script>
+  <script src="https://cdn.jsdelivr.net/npm/algoliasearch@4.0.0-beta.13/dist/algoliasearch-lite.umd.min.js"></script>
```

Comment:
I'm wondering if it'd make sense to change the `jsdelivr` and `unpkg` entries to `algoliasearch/lite`.

// @nunomaduro

Reply:
I think we are good point the CDN to the default build.